src/shared/tpm2-util.c

```diff
@@ -2345,7 +2345,7 @@ int tpm2_create_loaded(
         return 0;
 }
 
-int tpm2_marshal_private(const TPM2B_PRIVATE *private, void **ret, size_t *ret_size) {
+static int tpm2_marshal_private(const TPM2B_PRIVATE *private, void **ret, size_t *ret_size) {
```

Comment:
there's a trick here: they weren't used before, so in order to avoid the compiler error that static functions aren't used I simply exported them. And now that they are use I can unexport them. Seemed cleaner to me that way. Will add comment.

Reply:
ok, mention it in the commit message then, so that it's clearer